### ISSUE TITLE
ApplicationMapping leftover after deleting bound Application

### ISSUE THREAD

**Issue by kwiatekus** [kyma-project/kyma#6313]
**Description**

On my local minikube I have noticed a continous increase of memory consumption for console backend service and eventually a crash loop.

In the logs of the `console-backend-service` container I see the same log line repeated thousands of times per second:

```
 ... 
W1127 11:51:45.277247       1 app_service.go:241] Found ApplicationMapping "Application" in namespaces "test" but default with name "test" does not exists
W1127 11:51:45.324456       1 app_service.go:241] Found ApplicationMapping "Application" in namespaces "test" but default with name "test" does not exists
W1127 11:51:45.330157       1 app_service.go:241] Found ApplicationMapping "Application" in namespaces "test" but default with name "test" does not exists
W1127 11:51:45.336468       1 app_service.go:241] Found ApplicationMapping "Application" in namespaces "test" but default with name "test" does not exists
...
```

I had a `test` application defined, but I deleted it. Looks like there is a leftover of ApplicationMapping that was not cleared properly.

The logging of the log line is going on with huge pace that this could be the reason of OOM as it leads temporarily storing log lines in memory 

**Steps to reproduce**

 - Create Application and bind to a namespace
 - Delete application
 - observe the logs and memory consumption of console-backend-service

**Comment by kwiatekus** [comment]
Duplicates : https://github.com/kyma-project/console/issues/1425

**Comment by davyodom** [comment]
I think we faced similar issues.  Did you try to just manually delete the leftover ApplicationMappings?  Does this resolve the issue as a workaround?

**Comment by kwiatekus** [comment]
The sevirity of this issue decseased after we fixed proped web-socket link initialisation for console. Having this fixed (#6447) we longer see those log lines appearing is such huge pace.
So the OOM crash is no longer the case.

However, the remaining ApplicationMapping is still there. And will be handled in different ticket -> https://github.com/kyma-project/kyma/issues/3728
